.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           make test
         env:
           PXC_OPERATOR_VERSION: 1.16.1
-          PSMDB_OPERATOR_VERSION: 1.19.0
+          PSMDB_OPERATOR_VERSION: 1.19.1
```

Comment:
Leaving the comment to remember to update the go.mod dependency before merging. Now it's not possible because the release is not published yet and there is no such tag in psmdb

Reply:
Done